data/na/swagger/1-0-5.yaml

```diff
@@ -921,6 +923,25 @@ definitions:
         type: string
         maxLength: 20
         description: 'The Visa Checkout payment request ID.'
+  3d_secure:
+    description: "Payment with a 3D Secure Session Data token.  Reference a previously proessed 3D Secure transaction 
+      processed through the EMV 3DS API.  This will apply the associated card number, card expiry, and 3DSecure results 
```

Comment:
need a space in '3D Secure' for consistency

Reply:
updated